Copilot/README.md

```diff
@@ -0,0 +1,348 @@
+>     - A sandbox Microsoft 365 tenant with Copilot (available in limited preview through [TAP membership](https://developer.microsoft.com/microsoft-365/tap)).
+>     - An [eligible Microsoft 365 or Office 365 production environment](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#customers-with-existing-microsoft-365-and-copilot-licenses) with a Copilot for Microsoft 365 license.
+
+## Prerequisites
+1. Your M365 tenant is enabled for the private preview functionality. This is the tenant you will register your SPE app with. If it's not enabled already, you can request access to the private preview using this link: https://aka.ms/specopilotpreview. Our team will review the request and if it's approved, we will reach out to you within a week.
+2. A Copilot license enabled for at least one user in that tenant. You have two ways to get a developer environment for Copilot:
+- [A sandbox Microsoft 365 tenant](https://developer.microsoft.com/en-us/microsoft-365/tap) with Copilot (available in limited preview through TAP membership). 
+- [An eligible Microsoft 365 or Office 365 production environment](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#customers-with-existing-microsoft-365-and-copilot-licenses
+) with a Copilot for Microsoft 365 license.
+
+3. A working React SharePoint Embedded application, written in TypeScript.
+    - If you are not using TypeScript, please see the Appendix below for steps to take to get this running in your app.
+    - To get started with SharePoint Embedded, visit https://aka.ms/start-spe
+    - Then, if you want get started with a sample React SPE app, visit [spe-azurefunction](https://github.com/microsoft/SharePoint-Embedded-Samples/tree/main/Samples/spa-azurefunction)
+4. Your Container Type configuration for `DiscoverabilityDisabled` must be set to `false`
+    - Visit [Configuring Container Types](https://learn.microsoft.com/en-us/sharepoint/dev/embedded/concepts/app-concepts/containertypes#configuring-container-types) to learn how to set this configuration
+
+## Quick Start
+
+### 1. Use the `SharePoint-Embedded-Samples\Samples\spe-typescript-react-azurefunction` Application
+
+Naviagate to the `SharePoint-Embedded-Samples\Samples\spe-typescript-react-azurefunction\react-client\src\components\ChatSidebar.tsx` file and adjust the return statement of this function by removing the commented out react function component code.
+```typescript
+export const ChatSidebar: React.FunctionComponent = () => {
+
+    // return (<>
+
+    // </>);
+
+    const [chatAuthProvider, setChatAuthProvider] = React.useState<ChatAuthProvider | undefined>();
+...
+```
+
+Then navigate to the `Q:\src\SPE-Demo\SharePoint-Embedded-Samples\Samples\spe-typescript-react-azurefunction\react-client\src\routes\App.tsx` file and set the toggle to true.
```

Comment:
This is also confusing. Just ask them to look for this line:
```ts
  const [showSidebar, setShowSidebar] = useState<boolean>(false);
```
and replace it with this line
```ts
  const [showSidebar, setShowSidebar] = useState<boolean>(true);
```